src/plugins/blocksmith/components/document-block-modal/index.jsx

```diff
@@ -60,9 +61,10 @@ class DocumentBlockModal extends React.Component {
     this.state = {
       activeStep: 0,
       aside: block.get('aside'),
+      aspectRatio: block.get('aspect_ratio') || AspectRatioEnum.create('auto'),
```

Comment:
@efjacobson one, a default isn't "really" needed, however, if doing this, just use the enum property `AspectRatioEnum.AUTO`

Reply:
@gdbrown its not so much about a default value on the block - its also to hydrate the `Select` component. regardless, changed to `AUTO` way

Reply:
@efjacobson if we don't need to store a value at all I'd prefer that.  like if they didn't select anything just don't populate it.

Reply:
@gdbrown updated - the aspect ratio will now only be set if it is not AUTO